docker_test.go

```diff
@@ -271,6 +292,64 @@ func run(m *testing.M) int {
 		log.Fatalf("Could not connect to docker: %s", err)
 	}
 
+	// pulls an redis image, creates a container based on it and runs it
+	resourceRedis, err := pool.Run("redis", "alpine3.14", []string{"requirepass=secret"})
+	if err != nil {
+		log.Fatalf("Could not start resource: %s", err)
+	}
+	defer func() {
+		if err := pool.Purge(resourceRedis); err != nil {
+			log.Printf("Could not purge resource: %s \n", err)
+		}
+	}()
+	// exponential backoff-retry, because the application in the container might not be ready to accept connections yet
+	address = fmt.Sprintf("localhost:%s", resourceRedis.GetPort("6379/tcp"))
```

Comment:
A less generic name, maybe:
```suggestion
	redisHost = fmt.Sprintf("localhost:%s", resourceRedis.GetPort("6379/tcp"))
```
Would be better I think.